addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -24,6 +26,7 @@ var _states: Dictionary = {}
 var _inputs: Dictionary = {}
 var _latest_state: int = -1
 var _earliest_input: int
+var _started_at_tick: int = -1
```

Comment:
This variable ultimately ended up being unused, but its useful to keep so as to know when an avatar joined/started.